jobs/models.py

```diff
@@ -69,7 +68,81 @@ def set_expiration_date(self):
                 self.expiration_date = self.published_date + timedelta(60)
                 self.save()
```

Comment:
I'm not sure why we have the exact same code here for self.expiration_date and not self.expiration_date.  If there is an expiration date, we shouldn't be setting a new one (i.e. the second nested if statement (if self.ready_to_publish and self.expiration_date shouldn't be there at all)